dali_tf_plugin/dali_tf_plugin_utils.py

```diff
@@ -112,7 +112,7 @@ def get_tf_build_flags():
             tf_lflags=" ".join(["-L" + tensorflow_path, "-ltensorflow_framework"])
 
     if tf_cflags is '' and tf_lflags is '':
-        raise ImportError('Could not find Tensorflow. Tensorflow must be installed before installing nvidia-dali-tf-plugin')
+        raise ImportError('Could not find Tensorflow. Tensorflow must be installed before installing NVIDIA DALI TF plugin')
```

Comment:
Do we want to use capitalized name?

Reply:
What do you suggest?

Reply:
I'm just asking about using `NVIDIA DALI TF` instead of `nvidia-dali-tf`. If you want to change it here, the docs need to be updated as well to have consistent notation.

Reply:
`nvidia-dali-tf-plugin` is the name of the package in python/pip. Now it has a different name, depending on the CUDA version so I am referring to it not with the literal package name but with its name in English.

Reply:
ok

Reply:
I could rename to NVIDIA DALI TensorFlow plugin. Would that be better?

Reply:
I'm good either way. Up to you.